composer.json

```diff
@@ -67,17 +69,12 @@
     "autoload": {
         "psr-4": {
             "Sonata\\CoreBundle\\": "src/CoreBundle/",
-            "Sonata\\Form\\": "src/Form/",
-            "Sonata\\Serializer\\": "src/Serializer/",
-            "Sonata\\Twig\\": "src/Twig/"
+            "Sonata\\Serializer\\": "src/Serializer/"
```

Comment:
Do we need to keep this?

Reply:
Serializer was moved to form-extensions. We should keep sonata/serializer to keep BC.